frontend/src/features/admin/access-request/__snapshots__/AccessRequestContainer.test.tsx.snap

```diff
@@ -352,15 +352,27 @@ exports[`AccessRequestContainer component makes a request based on the access re
             </option>
             <option
               class="option"
-              value="5"
+              value="6"
             >
-              Real Estate Analyst
+              Finance
```

Comment:
Lot of updates to snapshots because I updated the mock code table lookups with up to date values